bool[] occupancy
geometry_msgs/Point[] positions